### PR TITLE
docs: add classifier docs

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
- [ ] will update this when we add one classifier in hub

### CODE REVIEW

File: chapters/extend/classifier.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+==================
```
Comment (CatStark) [inline chapters/extend/classifier.rst]: Can you add summary of the topics that will be covered? something like this

.. contents:: Table of Contents
    :depth: 2

Comment (Yongxuanzhang) [reply chapters/extend/classifier.rst]: 👍

**Comment by Yongxuanzhang** [comment]
Since there are no classifiers in Jina hub. We could provide an overview of it to illustrate our intentions and welcome contributions.
https://deploy-preview-27--jina-docs.netlify.app/chapters/classifier.html

File: chapters/classifier.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Classifier in Jina
+==================
+Classifier allows you to perform classification and regression based on given input and have the predicted hard/soft labels as output.
+
+.. contents:: Table of Contents
+    :depth: 3
+
+Motivation
+------------------------------------------
+:class:`Classifier` can be used for document meta data enrichment. :class:`Classifier` may help to generate and add tags to Documents automatically
```
Comment (bwanglzu) [inline chapters/classifier.rst:11]: i think this reference will generate a warning message when you `make build`, `jina.executors.classifiers.BaseClassifier` should work. Or you simply use `Classifier`

File: chapters/classifier.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+to serve as a smart annotation method. It can enable future features such as smart sub-flow routing.
```
Comment (bwanglzu) [inline chapters/classifier.rst:12]: i think it's worth to mention we're not using it at the moment with a `note` section

**Review by bwanglzu — COMMENTED** [review]
I'm just curious, is it a good moment to include Classifier into documentation since we're not using it internally @FionnD

**Comment by bwanglzu** [comment]
maybe something good to do is to add Classifier to [Glossary](https://docs.jina.ai/chapters/glossary/glossary.html)

**Pull request closed without merging**